### PR TITLE
Teamkk

### PR DESCRIPTION
**Pull request by kerryhuang1**
Used react-schedule-selector to make the dragging functionality. Ultimately decided to just implement a weekly view. Still fleshing out some details.

### CODE REVIEW

File: client/src/components/Calendar/TODO.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+EVERYTHING THAT STILL NEEDS TO BE DONE:
```
Comment (bdeleonardis1) [inline client/src/components/Calendar/TODO.txt]: Like I mentioned for the other thing, let's just file github issues for all these and delete the file.

File: client/src/index.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Calendar from "./components/Calendar/Calendar";
 
-import App from "./App.js"
+function App() {
```
Comment (bdeleonardis1) [inline client/src/index.js]: Let's revert this to the old index.js file. You already have the calendar display at the /calendar link, no reason to do it here too, right?

File: client/src/App.js [hunk 3]
```diff
@@ -3,24 +3,24 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
...
 import Login from './components/Login/Login';
+import Calendar from './components/Calendar';
 
 import './App.css'
 
 class App extends React.Component {
   constructor(props) {
     super(props);
   }
-
```
[the remark this answers is not in the document]
Comment (kerryhuang1) [reply client/src/App.js]: Not sure what this means

**Comment by kerryhuang1** [comment]
RAFL @bdeleonardis1 @leebianca 

Summary: 
- Removed package2.json, TODO.txt 
- Reverted index.js to starter code
- Reformatted with prettier
- Removed all comments

**Pull request merged**